dev/fattest.simplicity/src/componenttest/rules/repeater/MicroProfileActions.java

```diff
@@ -390,10 +432,12 @@ public class MicroProfileActions {
     public static final FeatureSet MP61 = new FeatureSet(MP61_ID, MP61_FEATURE_SET, EEVersion.EE10);
     public static final FeatureSet MP70_EE10 = new FeatureSet(MP70_EE10_ID, MP70_EE10_FEATURE_SET, EEVersion.EE10);
     public static final FeatureSet MP70_EE11 = new FeatureSet(MP70_EE11_ID, MP70_EE11_FEATURE_SET, EEVersion.EE11);
+    public static final FeatureSet MP71_EE10 = new FeatureSet(MP71_EE10_ID, MP71_EE10_FEATURE_SET, EEVersion.EE10);
+    public static final FeatureSet MP71_EE11 = new FeatureSet(MP71_EE11_ID, MP71_EE11_FEATURE_SET, EEVersion.EE11);
     public static final FeatureSet MP70_EE11_APP_MODE = new FeatureSet(MP70_EE11_APP_MODE_ID, MP70_EE11_FEATURE_SET, EEVersion.EE11);
 
-    //All MicroProfile FeatureSets - must be descending order
-    private static final FeatureSet[] ALL_SETS_ARRAY = { MP70_EE11, MP70_EE10, MP61, MP60, MP50, MP41, MP40, MP33, MP32, MP30, MP22, MP21, MP20, MP14, MP13, MP12, MP10 };
+    //All MicroProfile FeatureSets - must be descending order 
+    private static final FeatureSet[] ALL_SETS_ARRAY = { MP70_EE11, MP70_EE10, MP61, MP60, MP50, MP41, MP40, MP33, MP32, MP30, MP22, MP21, MP20, MP14, MP13, MP12, MP10 }; //TODO : Add MP71
```

Comment:
Is there some reason we can't add MP71 here yet?

Reply:
This repeat action already contains MP7.0 with both Jakarta versions. I don't think full tests need to run MP 7.1 yet if the new features haven't been even created yet. We would just be unnecessarily repeating the tests with the same features as MP 7.0

Reply:
I guess we could add it in now but I didn't see the value in it if MP 7.0 and MP 7.1 contain the same subset of features at the moment. I was worried about unnecessarily repeating tests

Reply:
We do need to add the new feature set to this list.

We mostly use the `ALL` set to calculate the full set of all features which we can replace when doing a repeat actions. This is unfortunately necessary due to the way the original FeatureReplacementAction was designed.

Several of the `repeat` methods in this class use it and you can follow through the method calls to see how it ultimately gets used.

Reply:
Technically, you don't need it at the moment because `MP70` and `MP71` are identical, but there's no reason not to update it now.

I don't think there's anywhere where we use this list as a list of configurations to test so I don't think it will result in any additional repeats running.